### PR TITLE
Curated suggestions highlights when they have a CurationNotice

### PR DESCRIPTION
**Pull request by Raemon**
This PR
* makes it more obvious when a curation suggestion has a CurationNotice already
* shows you the curation notice when you hover over it
* lets you click on the "publish and curate button" on the hoverover.
* attempts to make it so you can't accidentally doubleclick it, but, I haven't tested it yet
* separately, makes the title of the post on CurationNotice a link to the post

image

image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208861793574544) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by oetherington — COMMENTED** [review]
No objections from EAF (but untested since we don't use this)

File: packages/lesswrong/lib/collections/posts/schema.tsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ import { stableSortTags } from '../tags/helpers';
 import { getLatestContentsRevision } from '../revisions/helpers';
 import { marketInfoLoader } from './annualReviewMarkets';
 import { getUnusedSlugByCollectionName } from '@/lib/helpers';
+import { CurationNotices } from '../curationNotices/collection';
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/posts/schema.tsx:45]: remove this import (importing collection files into other collection files/schemas could cause problems).  you in any case have access to it from the ResolverContext

File: packages/lesswrong/lib/collections/posts/schema.tsx [hunk 2]
```diff
@@ -584,6 +585,7 @@ const schema: SchemaType<"Posts"> = {
       resolver: (revisionsField) => revisionsField("wordCount"),
     }),
   }),
+
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/posts/schema.tsx:588]: nit: remove

**Review by b0b3rt — APPROVED** [review]
approved, but please remove the extraneous import first

**Pull request merged**